UPDATING.md

```diff
@@ -43,6 +43,7 @@ assists people when migrating to a new version.
   make it more clear which envrionment your are in.
   `SUPERSET_ENV=production` and `SUPERSET_ENV=development` are the two
   supported switches based on the default config.
+- [19242](https://github.com/apache/superset/pull/19242): Adhoc subqueries were disabled by default for security reasons. To enable them, set `ALLOW_ADHOC_SUBQUERY` to `True` in your `superset_config.py`.
```

Comment:
I think we can scratch the `in your …` part. I gather (per other updates) this is implicitly implied and the actual filename could vary by deployment.

Reply:
"were" may be read as "have been in the past but are now something else". Could we reword that to "are now" or something more explicit?

Reply:
Done.